sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

```diff
@@ -115,21 +129,27 @@ export interface KeyVaultRoleAssignmentProperties {
 export interface KeyVaultRoleAssignmentPropertiesWithScope {
     principalId: string;
     roleDefinitionId: string;
-    scope?: RoleAssignmentScope;
+    scope?: KeyVaultRoleAssignmentScope;
 }
 
+// @public
+export type KeyVaultRoleAssignmentScope = "/" | "/keys" | string;
+
 // @public
 export interface KeyVaultRoleDefinition {
     assignableScopes: string[];
     description: string;
     readonly id: string;
+    readonly kind: string;
     readonly name: string;
     permissions: KeyVaultPermission[];
     roleName: string;
     roleType: string;
-    readonly type: string;
 }
 
+// @public
+export type KeyVaultRoleScope = "/" | "/keys";
```

Comment:
This and the RoleAssignment scope look way too similar. Would it make sense to unify them? if not, why?

Reply:
Aren't they the same? I believe that is how it is defined in the swagger.

Reply:
Good catch! I'll fix this shortly.

Reply:
@christothes @sadasant I realized something isn't quite right - In the swagger it looks like scopes can only be `/` or `/keys` (https://github.com/Azure/azure-rest-api-specs/blame/master/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/rbac.json#L682-L705) 

But our current definition for `KeyVaultRoleAssignmentScope` allows `/`, `/keys` or any string - shouldn't the swagger also support something like `/keys/<id>` for individual keys?

Reply:
At least in .NET, when an enum is modeled as string, it can be used as "strongly typed" for the known values or a string of any value can be used by constructing a new instance of the generated "extensible enum" type with any arbitrary string.

Reply:
Thanks @christothes!  I'll investigate why the generated code isn't an extensible enum on our side